src/gc/gcdiag.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __GCDIAG_H__
+#define __GCDIAG_H__
+
+typedef void (* record_surv_fn)(uint8_t*, uint8_t*, ptrdiff_t, size_t, BOOL, BOOL);
```

Comment:
`BOOL` -> `bool`. (The GC interfaces should be using the native C++ primitive types, not the Windows-specific ones.)